eessi_container.sh

```diff
@@ -0,0 +1,458 @@
+  echo "                          [default: docker://ghcr.io/eessi/build-node:debian11]"
+  echo "  -h | --help           - display this usage information [default: false]"
+  echo "  -g | --storage DIR    - directory space on host machine (used for"
+  echo "                          temporary data) [default: 1. TMPDIR, 2. /tmp]"
+  echo "  -m | --mode MODE      - with MODE==shell (launch interactive shell) or"
+  echo "                          MODE==run (run a script) [default: shell]"
+  echo "  -r | --repository CFG - configuration file or identifier defining the"
+  echo "                          repository to use [default: EESSI-pilot via"
+  echo "                          container configuration]"
+  echo "  -u | --resume DIR/TGZ - resume a previous run from a directory or tarball,"
+  echo "                          where DIR points to a previously used tmp directory"
+  echo "                          (check for output 'Using DIR as tmp ...' of a previous"
+  echo "                          run) and TGZ is the path to a tarball which is"
+  echo "                          unpacked the tmp dir stored on the local storage space"
+  echo "                          (see option --storage above) [default: not set]"
+  echo "  -s | --save DIR/TGZ   - save contents of tmp directory to a tarball in"
+  echo "                          directory DIR or provided with the fixed full path TGZ"
+  echo "                          when a directory is provided, the format of the"
+  echo "                          tarball's name will be {REPO_ID}-{TIMESTAMP}.tgz"
+  echo "                          [default: not set]"
+  echo
+  echo " If value for --mode is 'run', the SCRIPT provided is executed."
+  echo
+  echo " FEATURES/OPTIONS to be implemented:"
+  echo "  -d | --dry-run           -  run script except for executing the container,"
+  echo "                              print information about setup [default: false]"
+  echo "  -i | --info              -  display configured repositories [default: false]"
+  #echo "  -x | --http-proxy URL    -  provides URL for the env variable http_proxy"
+  #echo "                              [default: not set]"
+  #echo "  -y | --https-proxy URL   -  provides URL for the env variable https_proxy"
+  #echo "                              [default: not set]"
+}
+
+# set defaults for command line arguments
+ACCESS="ro"
+CONTAINER="docker://ghcr.io/eessi/build-node:debian11"
+DRY_RUN=0
+INFO=0
+STORAGE=
+MODE="shell"
+REPOSITORY="EESSI-pilot"
+RESUME=
+SAVE=
+#HTTP_PROXY=
+#HTTPS_PROXY=
+RUN_SCRIPT_AND_ARGS=
+
+POSITIONAL_ARGS=()
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -a|--access)
+      ACCESS="$2"
+      shift 2
+      ;;
+    -c|--container)
+      CONTAINER="$2"
+      shift 2
+      ;;
+    -d|--dry-run)
+      DRY_RUN=1
+      shift 1
+      ;;
+    -g|--storage)
+      STORAGE="$2"
+      shift 2
+      ;;
+    -h|--help)
+      display_help
+      exit 0
+      ;;
+    -i|--info)
```

Comment:
maybe rename to `--verbose`?

Reply:
fixed in [09feff7](https://github.com/EESSI/software-layer/pull/216/commits/09feff74d11d4cf14393cde33abd0672eb9f3bcf)